### ISSUE TITLE
[libinput]: slow mousewheel scrolling for qml widgets when using libinput

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.13#42]

**Comment by davidedmundson** [comment]
Found it.

In summary because libinput tries to be smart, and make mouse wheel events and trackpad scrolls look the same to apps. Qt tries to be smart and treat mouse wheels and trackpads differently with different speeds. That no longer works.

I've filed a Qt bug where the root cause is.
https://bugreports.qt.io/browse/QTBUG-59261

I'll make a Qt patch for Netrunner/Maui that hides the worst of the problem immediately.

On QtQuickControls 2 the problem isn't so bad. It isn't technically the same as the speed on QWidgets, but it will scroll twice as fast as on QQC1. I think for main Plasma that will be the quickest route to take (given we want to do that anyway)
